src/rgw/rgw_op.h

```diff
@@ -104,6 +105,8 @@ RGWOp() : s(nullptr), dialect_handler(nullptr), store(nullptr),
   virtual uint32_t op_mask() { return 0; }
 
   virtual int error_handler(int err_no, string *error_content);
+
+  boost::function<void()> dump_access_control_f();
```

Comment:
i don't understand the motivation for this `boost::function` part.. why can't `end_header()` continue to call `dump_access_control()` directly?

the commit message says `I would like the http routines to known less about "op" structures`, but Op and Handler are two of the most basic parts of our rest framework